Sources/Query.swift

```diff
@@ -8,159 +8,507 @@
 
 import Foundation
 import Interstellar
+import CoreLocation
 
+/// The available URL parameter names for queries; used internally by the various `Contentful.Query` types.
+/// If you are constructing Dictionary's to pass into `Client` "fetch" methods, instead of taking advantage of "fetch" methods
+/// that take `Query` types, use these static variables to avoid typos in query construction.
```

Comment:
Hmm, the last sentence sounds a bit clunky to me. Can we use something like `Use the static variables for typo safety. It is recommended to use the _fetch_ methods to your advantages.`?